src/main/java/seedu/address/logic/commands/AddAppointmentCommand.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_APPOINTMENT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_END;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_START;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.appointment.Appointment;
+import seedu.address.model.person.Person;
+
+
+/**
+ * Add the appointment details of an existing person in the address book.
+ */
+public class AddAppointmentCommand extends Command {
+    public static final String COMMAND_WORD = "appointment";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Add appointment details of the patient specified "
+            + "by the index number used in the displayed patient list.\n"
+            + "Parameters: "
+            + "INDEX (must be a positive integer) "
+            + PREFIX_APPOINTMENT + "DESCRIPTION "
+            + PREFIX_START + "START DATE TIME "
+            + PREFIX_END + "END DATE TIME "
+            + "\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_APPOINTMENT + " Surgery "
+            + PREFIX_START + " 21-10-2024-10-30 "
+            + PREFIX_END + " 21-10-2024-11-00 ";
```

Comment:
I think can include the outline DD-MM-YYYY-HH-MM as a guide for users to reduce ambiguity on the formats of datetime 😃